### PR TITLE
debug message

### PR DESCRIPTION
**Pull request by nimitsarup**

### CODE REVIEW

File: sitemap/s3saver.go [hunk 1]
```diff
@@ -38,5 +40,6 @@ func (s *S3Saver) SaveFile(lang config.Language, body io.Reader) error {
 	if err != nil {
 		return fmt.Errorf("failed to upload file to s3: %w", err)
 	}
+	log.Info(context.Background(), fmt.Sprintf("saved key [%s], bucket [%s]", k, s.bucket))
```
Comment (cookel2) [inline sitemap/s3saver.go:43]: I don't think that the key should be logged due to security?

Comment (nimitsarup) [reply sitemap/s3saver.go:43]: the key is just the language english/welsh :)

Comment (cookel2) [reply sitemap/s3saver.go:43]: ah ok, thanks

**Review by cookel2 — APPROVED** [review]
LGTM

**Pull request merged**